tests/integration/targets/azure_rm_gallery/tasks/main.yml

```diff
@@ -343,6 +343,46 @@
     that:
       - output.changed
 
+- name: Create or update a simple gallery Image Version.
+  azure_rm_galleryimageversion:
+    resource_group: "{{ resource_group }}"
+    gallery_name: myGallery{{ rpfx }}
+    gallery_image_name: myImage
+    name: 10.1.3
+    location: eastus
+    publishing_profile:
+      end_of_life_date: "2050-10-01t00:00:00+00:00"
+      exclude_from_latest: true
+      replica_count: 3
+      storage_account_type: Standard_LRS
+      target_regions:
+        - name: eastus
+          regional_replica_count: 1
+          encryption:
+            data_disk_images:
+              - disk_encryption_set_id: "{{ des_results.state.id }}"
+            os_disk_image:
+              disk_encryption_set_id: "{{ des_results.state.id }}"
+        - name: westus
+          regional_replica_count: 2
+          encryption:
+            data_disk_images:
+              - disk_encryption_set_id: "{{ des_results.state.id }}"
+            os_disk_image:
+              disk_encryption_set_id: "{{ des_results.state.id }}"
+          storage_account_type: Standard_ZRS
+    storage_profile:
```

Comment:
Parameter definition sequence: storage_profile -> os_disk -> source -> uri
```e.g.

383     storage_profile:
384       os_disk:
385         source:
386           resource_group: "{{ resource_group }}"
387           storage_account: '{{ vms_output.vms[0].storage_account_name }}'
388           uri: 'https://{{ vms_output.vms[0].storage_account_name }}.blob.core.windows.net/{{ vms_output.vms[0].stor    age_container_name }}/{{ vms_output.vms[0].storage_blob_name }}'

Reply:
@jeremycline The output variable on line 377 should be the one obtained by the VM on line 108. But lines 108 through 380 also register other output variables. So should pay attention to 120, 377, 378 this variable value acquisition! Thank you very much!

Reply:
Ah I see, it's being shadowed. My Ansible is very rusty, sorry. I _think_ I've fixed it.